src/services/NurseryInventoryService.ts

```diff
@@ -131,22 +131,52 @@ const searchInventory = async ({
   facilityIds,
   query,
 }: SearchInventoryParams): Promise<SearchResponseElement[] | null> => {
-  const params: SearchNodePayload = {
-    prefix: 'inventories',
-    fields: [...BE_SORTED_FIELDS, 'species_commonName', 'totalQuantity(raw)'],
-    sortOrder: searchSortOrder ? [searchSortOrder] : undefined,
-    search: {
-      operation: 'and',
-      children: [
-        {
-          operation: 'field',
-          field: 'organization_id',
-          values: [organizationId],
-        },
+  let params: SearchNodePayload;
+  const forSpecificFacilities = !!facilityIds && !!facilityIds.length;
+  if (forSpecificFacilities) {
+    params = {
+      prefix: 'inventories.facilityInventories',
+      fields: [
+        'species_id',
+        'species_scientificName',
+        'species_commonName',
+        'facility_name',
+        'germinatingQuantity(raw)',
+        'readyQuantity(raw)',
+        'notReadyQuantity(raw)',
+        'totalQuantity(raw)',
       ],
-    },
-    count: 0,
-  };
+      sortOrder: searchSortOrder ? [searchSortOrder] : undefined,
+      search: {
+        operation: 'and',
+        children: [
+          {
+            operation: 'field',
+            field: 'organization_id',
+            values: [organizationId],
+          },
+        ],
+      },
+      count: 0,
+    };
+  } else {
+    params = {
+      prefix: 'inventories',
+      fields: [...BE_SORTED_FIELDS, 'species_commonName', 'totalQuantity(raw)'],
+      sortOrder: searchSortOrder ? [searchSortOrder] : undefined,
+      search: {
+        operation: 'and',
+        children: [
+          {
+            operation: 'field',
+            field: 'organization_id',
+            values: [organizationId],
+          },
+        ],
+      },
+      count: 0,
+    };
+  }
```

Comment:
suggestion, since the only difference between those params are the prefix and fields, we can maybe do something like this

```
  const params: SearchNodePayload = {
    prefix: forSpecificFacilities ? 'inventories.facilityInventories' : 'inventories',
    fields: forSpecificFacilities ? FACILITY_SPECIFIC_FIELDS : INVENTORY_FIELDS, (maybe extract them into a constant)
    sortOrder: searchSortOrder ? [searchSortOrder] : undefined,
    search: {
        operation: 'and',
        children: [
          {
            operation: 'field',
            field: 'organization_id',
            values: [organizationId],
          },
        ],
    },
    count: 0,
  };
```

Thoughts?